.eslintrc.js

```diff
@@ -62,6 +62,7 @@ module.exports = {
         "prefer-const": "error",
         "quotes": ["error", "double"],
         "semi": "off",
-        "react/prop-types": "off"
+        "react/prop-types": "off",
+        "no-async-promise-executor": "off"
```

Comment:
I kinda like this, so you don't forget to await stuff :(

Reply:
No, this was something else completely but don't need it anymore